README.md

```diff
@@ -1,5 +1,8 @@
-# Blood Magic: Alchemical Wizardry [![](http://cf.way2muchnoise.eu/full_blood-magic_downloads.svg)](https://minecraft.curseforge.com/projects/blood-magic) [![Discord](https://img.shields.io/discord/259683256348311552.svg?colorB=7289DA&logo=data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAHYAAABWAgMAAABnZYq0AAAACVBMVEUAAB38%2FPz%2F%2F%2F%2Bm8P%2F9AAAAAXRSTlMAQObYZgAAAAFiS0dEAIgFHUgAAAAJcEhZcwAACxMAAAsTAQCanBgAAAAHdElNRQfhBxwQJhxy2iqrAAABoElEQVRIx7WWzdGEIAyGgcMeKMESrMJ6rILZCiiBg4eYKr%2Fd1ZAfgXFm98sJfAyGNwno3G9sLucgYGpQ4OGVRxQTREMDZjF7ILSWjoiHo1n%2BE03Aw8p7CNY5IhkYd%2F%2F6MtO3f8BNhR1QWnarCH4tr6myl0cWgUVNcfMcXACP1hKrGMt8wcAyxide7Ymcgqale7hN6846uJCkQxw6GG7h2MH4Czz3cLqD1zHu0VOXMfZjHLoYvsdd0Q7ZvsOkafJ1P4QXxrWFd14wMc60h8JKCbyQvImzlFjyGoZTKzohwWR2UzSONHhYXBQOaKKsySsahwGGDnb%2FiYPJw22sCqzirSULYy1qtHhXGbtgrM0oagBV4XiTJok3GoLoDNH8ooTmBm7ZMsbpFzi2bgPGoXWXME6XT%2BRJ4GLddxJ4PpQy7tmfoU2HPN6cKg%2BledKHBKlF8oNSt5w5g5o8eXhu1IOlpl5kGerDxIVT%2BztzKepulD8utXqpChamkzzuo7xYGk%2FkpSYuviLXun5bzdRf0Krejzqyz7Z3p0I1v2d6HmA07dofmS48njAiuMgAAAAASUVORK5CYII%3D)](https://discord.gg/VtNrGrs)
-
+# Blood Magic: Alchemical Wizardry 
+[![Curseforge](https://img.shields.io/curseforge/dt/224791.svg?style=for-the-badge&label=Curseforge&logo=curseforge&color=f16436)](https://minecraft.curseforge.com/projects/blood-magic)
+[![Modrinth](https://img.shields.io/modrinth/dt/PbNc6qBY.svg?style=for-the-badge&label=Modrinth&logo=modrinth&color=1bd96a)](https://modrinth.com/mod/blood-magic)
+[![Discord](https://img.shields.io/discord/259683256348311552.svg?style=for-the-badge&label=Discord&logo=discord&color=404eed)](https://discord.gg/VtNrGrs)
+[![License: CC BY 4.0](https://img.shields.io/badge/License-CC_BY_4.0-lightgrey.svg?style=for-the-badge)](https://creativecommons.org/licenses/by/4.0/)
 ### Gruesome? Probably. Worth it? Definitely!
 ### [Downloads](http://minecraft.curseforge.com/projects/blood-magic/files)
```

Comment:
Duplicate CurseForge link down here?

Reply:
The first one is a badge. The second one is a plain download link. This was how it was earlier. You can see [this](https://github.com/mystchonky/BloodMagic/tree/licenses) to check how it looks